.cookiecutter/includes/setuptools/install_requires

```diff
@@ -1,2 +1,3 @@
-jsonschema
+jsonschema>=4.18
```

Comment:
We have to apply a minimum, as this is the first time this way of doing things is supported.